app/page.tsx

```diff
@@ -21,50 +22,13 @@ export default function Chat() {
 
 
   return (
-    <main className="mx-auto w-full p-24 flex flex-col">
+    <main className="mx-auto w-full flex flex-col">
       <div className="p4 m-4">
-        <div className="flex flex-col items-center justify-center space-y-8 text-white">
-          <div className="space-y-2">
-            <h2 className="text-3xl font-bold">Story Telling App</h2>
-            <p className="text-zinc-500 dark:text-zinc-400">
-              Customize the story by selecting the genre and tone.
-            </p>
-          </div>
+        <div className="flex flex-col justify-center space-y-8 text-white">
 
-          <CharacterTable
-            characters={characters}
-            setCharacters={setCharacters}
-          />
+          <Design characters={characters}
+            setCharacters={setCharacters}></Design>
           
-          {characters.length === 0 ? (
```

Comment:
I was worried about this logic part missing from the code. I mean to refuse to generate a story if there are no characters defined.

But I see the button is hidden if there are no characters, so it's more simple this way.

I accept, thank you! 🙏🏼